src/tests/GC/Stress/Framework/ReliabilityFramework.cs

```diff
@@ -689,7 +689,7 @@ private void TestStarter()
 
                 if (startTest)
                 {
-                    _logger.WriteToInstrumentationLog(_curTestSet, LoggingLevels.TestStarter, String.Format("Looking for test to start..."));
+                    // _logger.WriteToInstrumentationLog(_curTestSet, LoggingLevels.TestStarter, String.Format("Looking for test to start..."));
```

Comment:
Let's just delete these commented out lines altogether. Version control can be used to track the history of them.

Reply:
Option 1 seems to be the most viable for this - in no circumstances would we want unnecessary frequent log messages that aren't useful. As a follow up, we will be working on making the logging in the case a test can't run more descriptive and certainly less frequent.